libs/editor/libs/editor-common/assets/ZSSRichTextEditor.js

```diff
@@ -924,24 +924,42 @@ ZSSEditor.replaceLocalImageWithRemoteImage = function(imageNodeIdentifier, remot
         ZSSEditor.markImageUploadDone(imageNodeIdentifier);
         var joinedArguments = ZSSEditor.getJoinedFocusedFieldIdAndCaretArguments();
         ZSSEditor.callback("callback-input", joinedArguments);
-
+        image.onerror = null;
+        image.classList.add("image-loaded");
     }
 
     image.onerror = function () {
-        // Even on an error, we swap the image for the time being.  This is because private
-        // blogs are currently failing to download images due to access privilege issues.
-        //
-        imageNode.attr('src', image.src);
         imageNode.addClass("wp-image-" + remoteImageId);
         ZSSEditor.markImageUploadDone(imageNodeIdentifier);
         var joinedArguments = ZSSEditor.getJoinedFocusedFieldIdAndCaretArguments();
         ZSSEditor.callback("callback-input", joinedArguments);
-
+        // Try to reload the image on error.
+        ZSSEditor.tryToReload(image, 1);
     }
 
     image.src = remoteImageUrl;
 };
 
+ZSSEditor.reloadImage = function(node, nCall) {
+    if (node.classList.contains("image-loaded")) {
+        return;
+    }
+    console.log("Reloading image:", node, nCall);
+    node.onerror = tryToReload(node, nCall + 1);
```

Comment:
`tryToReload` needs to be `ZSSEditor.tryToReload`, otherwise we get a `ReferenceError`.